index.html

```diff
@@ -0,0 +1,160 @@
+                <!-- radio section -->
+                <section class="sec2-form">
+                    <ul>
+                        <li>
+                            <p>Would you recommend freeCodeCamp to a friend?</p>
+                        </li>
+                        <li class="radio_id">
+                            <input type="radio" id="Definitely" name="name" value="definitely">
+                            <label for="Definitely"> Definitely</label>
+                        </li>
+                        <li class="radio_id">
+                            <input type="radio" id="Maybe" name="name" value="Maybe">
+                            <label for="Maybe"> Maybe</label>
+                        </li>
+                        <li class="radio_id">
+                            <input type="radio" id="not_sure" name="name" value="Not sure">
+                            <label for="not_sure"> Not sure</label>
+                        </li>
+
+                    </ul>
+                </section>
+                <!-- selecting an option -->
+                <section class="sec-form">
+                    <ul>
+                        <li>
+                            <p>What is your favorite feature of freeCodeCamp?</p>
+                        </li>
+                        <li>
+                            <select name="" id="dropdown">
+                                <option value="" selected>Select an option</option>
+                                <option>Challenges</option>
+                                <option>Projects</option>
+                                <option>Community</option>
+                                <option>Open Source</option>
+                            </select>
+                        </li>
+
+                    </ul>
+                </section>
+                <!-- checkbox section -->
+                <section class="sec2-form">
+                    <ul>
+                        <li>
+                            <p>What would you like to see improved? (Check all that apply)</p>
+                            <div>
+                                <input type="checkbox" id="chk1" name="vehicle1" value="Bike">
+                                <label for="chk1"> Front-end Projects</label><br>
+
+                                <input type="checkbox" id="chk2" name="vehicle2" value="Car">
+                                <label for="chk2"> Back-end Projects</label><br>
+
+                                <input type="checkbox" id="chk3" name="vehicle3" value="Boat">
+                                <label for="chk3"> Data Visualization</label><br>
+
+                                <input type="checkbox" id="chk4" name="vehicle1" value="Bike">
+                                <label for="chk4"> Challenges</label><br>
+
+                                <input type="checkbox" id="chk5" name="vehicle2" value="Car">
+                                <label for="chk5"> Open Source Community</label><br>
```

Comment:
please try using a semantic tag instead of div like section, this improves the HTML structure and makes the page more accessible